spec/controllers/steps_controller_spec.rb

```diff
@@ -2,11 +2,17 @@
 
 RSpec.describe StepsController do
   context "when no applications have been started" do
-    it "redirects to the homepage" do
+    it "redirects to the homepage on a GET" do
       get :index
 
       expect(response).to redirect_to(root_path)
     end
+
+    it "redirects to the homepage on a PUT" do
+      put :update, params: { id: "" }
+
+      expect(response).to redirect_to(root_path)
+    end
```

Comment:
One question on this - should we include something in a flash message that tells then _why_ they were redirected back to the homepage? Right now we just send them back without any notice.

Reply:
flash message sounds nice, although the flash messages styles on our homepage are kinda wonky at the moment